test/extensions/filters/http/tap/tap_filter_integration_test.cc

```diff
@@ -233,6 +233,123 @@ name: tap
   verifyStaticFilePerTap(fmt::format(filter_config, getTempPathPrefix()));
 }
 
+TEST_P(TapIntegrationTest, StaticExtTapSinkUdp) {
```

Comment:
This should probably be added in a dedicated integration test file for tap_sinks/udp_sink.